src/DDTrace/Integrations/Symfony/SymfonySandboxedIntegration.php

```diff
@@ -0,0 +1,255 @@
+        $symfonyRequestSpan = null;
+        $isSymfony2 = false;
+
+        dd_trace_method(
+            'Symfony\Component\HttpKernel\HttpKernel',
+            '__construct',
+            function () use (&$appName, &$symfonyRequestSpan, $integration, &$isSymfony2) {
+                $tracer = GlobalTracer::get();
+                $scope = $tracer->getRootScope();
+                /** @var Span $symfonyRequestSpan */
+                $symfonyRequestSpan = $scope->getSpan();
+
+                if (
+                    defined('\Symfony\Component\HttpKernel\Kernel::VERSION')
+                        && Versions::versionMatches('2', \Symfony\Component\HttpKernel\Kernel::VERSION)
+                ) {
+                    $isSymfony2 = true;
+                    return false;
+                }
+
+                $appName = Configuration::get()->appName('symfony');
+                $symfonyRequestSpan->overwriteOperationName('symfony.request');
+                $symfonyRequestSpan->setTag(Tag::SERVICE_NAME, $appName);
+                $symfonyRequestSpan->setIntegration($integration);
+                $symfonyRequestSpan->setTraceAnalyticsCandidate();
+
+                return false;
+            }
+        );
+
+        dd_trace_method(
+            'Symfony\Component\HttpKernel\HttpKernel',
+            'handle',
+            function (
+                SpanData $span,
+                $args,
+                $response
+            ) use (
+                &$appName,
+                &$symfonyRequestSpan,
+                $integration,
+                &$isSymfony2
+            ) {
+                if ($isSymfony2) {
+                    // Disabled for symfony 2
+                    return false;
+                }
+
+                /** @var Request $request */
+                list($request) = $args;
+
+                $span->name = $span->resource = 'symfony.kernel.handle';
+                $span->service = $appName;
+                $span->type = Type::WEB_SERVLET;
+
+                $symfonyRequestSpan->setTag(Tag::HTTP_METHOD, $request->getMethod());
+                $symfonyRequestSpan->setTag(Tag::HTTP_URL, $request->getUriForPath($request->getPathInfo()));
+                $symfonyRequestSpan->setTag(Tag::HTTP_STATUS_CODE, $response->getStatusCode());
+
+                $route = $request->get('_route');
+                if (null !== $route && null !== $request) {
+                    $symfonyRequestSpan->setTag(Tag::RESOURCE_NAME, $route);
+                    $symfonyRequestSpan->setTag('symfony.route.name', $route);
+                }
+            }
+        );
+
+        dd_trace_method(
+            'Symfony\Component\EventDispatcher\EventDispatcher',
+            'dispatch',
+            function (SpanData $span, $args) use (&$appName, &$symfonyRequestSpan, $integration, &$isSymfony2) {
+                if ($isSymfony2) {
+                    // Disabled for symfony 2
+                    return false;
+                }
```

Comment:
All these symfony2 checks make me think we need to get a `dd_trace_untrace_method` or something so we can disable these once we know the Symfony version -- not necessary for this PR, of course.

Reply:
This might be a good idea, actually.